django/contrib/admin/static/admin/js/change_form.js

```diff
@@ -1,20 +1,17 @@
-/*global showAddAnotherPopup, showRelatedObjectLookupPopup showRelatedObjectPopup updateRelatedObjectLinks*/
-
-(function($) {
+(function() {
     'use strict';
-    $(document).ready(function() {
-        var modelName = $('#django-admin-form-add-constants').data('modelName');
-        $('body').on('click', '.add-another', function(e) {
-            e.preventDefault();
-            var event = $.Event('django:add-another-related');
-            $(this).trigger(event);
-            if (!event.isDefaultPrevented()) {
-                showAddAnotherPopup(this);
+    var inputTags = ['BUTTON', 'INPUT', 'SELECT', 'TEXTAREA'];
+    var modelName = document.getElementById('django-admin-form-add-constants').dataset.modelName;
```

Comment:
Are we not into using `const` / `let` yet ?

Reply:
That would be great to mover the JavaScript to `const` and `let`. I'll take a look at doing this.

Reply:
I made this change in #12767. If that change lands, I'll rebase this to also use `const`/`let`.